### DISCUSSION TITLE
ALTERAÇÃO DE CÓDIGO

### DISCUSSION THREAD

**Discussion by Stephanienogueira** [RamonSilva20/mapos#2195] [Q&A]
Boa tarde, alguém poderia me ajudar ?

Gostaria de alterar o código para que eu consiga inserir serviço sem valor. Mas quando eu for gerar um Orçamento eu consiga inserir esses serviços cadastrados e eu mesma coloque o valor do orçamento e/ou ordem de serviço.

Teria essa possibilidade ?

**Comment by FlaviusR** [comment]
Olá Stepahanie, tem sim.
Eu tenho serviços cadastrados com R$ 0,01.
Na hora de inserir na O.S. , é possivel editar o valor e a quantidade e entra direto na Orçamento com o valor atual.
